src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/HeapSizeIncreaseAction.java

```diff
@@ -18,12 +18,12 @@
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.AppContext;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions.ImpactVector.Dimension;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.rca.cluster.NodeKey;
-import java.util.ArrayList;
-import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
javax.annotation provides @Nonnull, is there a reason you're using this specific import?

Reply:
right, wanted to use Findbug's nonnull itself. We don't run checker framework checks yet.